server/main_server/src/main.cpp

```diff
@@ -604,7 +638,9 @@ Environment Variables:
         log_server::set_level(log_ns::get_level_from_config(irods::KW_CFG_LOG_LEVEL_CATEGORY_SERVER));
         log_ns::set_server_type("server");
         log_ns::set_server_zone(irods::get_server_property<std::string>(irods::KW_CFG_ZONE_NAME));
-        log_ns::set_server_hostname(boost::asio::ip::host_name());
+        log_ns::set_server_hostname(irods::get_server_property<std::string>(irods::KW_CFG_HOST));
```

Comment:
this isn't the logger fix... so not sure if you want to squash with the same commit.

Reply:
Logger fix? Not sure what you mean. Can you explain?

Reply:
probably won't matter once things are squashed.... this line was in a commit with only logger/stdout edits.   and this is about the host information from the server_config.json.

Reply:
Agreed. I'm still pulling things apart and gluing them back together so the PR is easier to consume.

Resolving.